Sources/ComposableArchitecture/Effect.swift

```diff
@@ -339,7 +339,7 @@ extension Publisher {
   /// into a result and then applying passed in function to it.
   ///
   /// This is a convenience operator for writing ``Effect/catchToEffect()`` followed by a
-  /// ``Effect/map(_:)-28ghh``.
```

Comment:
This was still the identifier in Xcode 13.2, so I guess some DocC bugs have been fixed!